app/assets/stylesheets/modules/_nav.styl

```diff
@@ -295,7 +295,7 @@ nav.ham-nav
 .course-nav__wrapper,
 .campaign-nav__wrapper
   height $nav_height
-  +below(desktop)
+  +below(desktop - 1)
```

Comment:
What's going on with the handful of 1-pixel changes?

Reply:
This is a borderline case, where the margin should be applied starting from 919px, so that is why the `-1` . At 920px, the margin gets applied but the mobile layout mode hasn't yet been applied

Reply:
Okay. A separate PR to clean up the hard-coding would be great.